package/MDAnalysis/analysis/dielectric.py

```diff
@@ -0,0 +1,148 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+
+r"""
+Dielectric --- :mod:`MDAnalysis.analysis.dielectric`
+===========================================================
+
+:Authors: Mattia Felice Palermo, Philip Loche
+:Year: 2019
+:Copyright: GNU Public License v2
+
+A tool to compute the average dipole moment :math:`M` and
+the static dielectric constant
+
+.. math::
+
+   \varepsilon = 1 + \frac{\langle M^2 \rangle - \langle M \rangle^2}
+                            {3 \varepsilon_ 0 V k_B T}
+
+for a system simulated in tin foil boundary conditions.
+"""
+
+from __future__ import division
```

Comment:
You need to also import `absolute_import` from `__future__` to make pylint happy.

Reply:
done